src/org/jitsi/impl/neomedia/rtcp/RTCPPacketParserEx.java

```diff
@@ -179,8 +179,12 @@ protected RTCPPacket parse(
 
                     return remb;
                 case FIRPacket.FMT:
-                    return new FIRPacket(senderSSRC, sourceSSRC);
+                    FIRPacket firPacket = new FIRPacket(senderSSRC, sourceSSRC);
+                    // read 32 bits of unused sequence number and reserved.
+                    in.readInt();
+                    return firPacket;
                 case PLIPacket.FMT:
+                    // PLIs have no feedback control information.
                     return new PLIPacket(senderSSRC, sourceSSRC);
```

Comment:
I really don't think this is the right place to have packet-type specific logic, and if I am reading 5104 correctly FIRs have one more word (SSRC).